about.html

```diff
@@ -12,62 +12,104 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
+    <div class="about-page">
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
 
+        <section class="tnav">
+            <header>
+            <nav>
+    
+              <img class="logo" src="img/lambda-black.png" alt="Lambda Logo.">  
+                   
+                <a href="index.html">Home</a>
+                <a href="#">About</a>
```

Comment:
Don't forget to update all of your links when you're working with multiple pages